docs/data-management/organization.qmd

```diff
@@ -56,9 +59,9 @@ project_name/
 ### `Data` folder
 
 * The `data` folder contains all project data sets. 
-* __Raw data is preserved in its own subfolder.__
+* __Raw data is preserved in its own subfolder.__ This folder also contains [raw data documentation readme files](../data-management/documentation.qmd#rawdatadoc).
```

Comment:
I personally always capitalize README but when it's not in a filename, it is probably more a matter of personal preference.
 
```suggestion
* __Raw data is preserved in its own subfolder.__ This folder also contains [raw data documentation README files](../data-management/documentation.qmd#rawdatadoc).
```